tensorflow/compiler/xla/service/hlo_instruction.cc

```diff
@@ -2282,6 +2280,14 @@ string PrintName(const string& name, bool print_ids) {
   }
 }
 
+string HloInstruction::NvtxNodeOpString() const { return metadata().op_type(); }
+
+string HloInstruction::NvtxNodeNameString() const {
```

Comment:
sanjoy@: how reliable / future-proof is it to generate the name in this way?

Reply:
Sorry did not notice this ping.  JFYI on github @sanjoy will notify me, but sanjoy@ won't.

I don't think this should be a function on `HloInstruction` since it isn't a core part of HLO (as you seem to indicate).  But it also doesn't have to be right?  This could live as a free function on a `nvtx_utils.h` / `nvtx_utils.cc` library?

Reply:
Let me change that real quick ;)

Reply:
Fixed in 927874e125e116338406920df1c62175e89d55d1 @sanjoy